freegames/hangman.py

```diff
@@ -0,0 +1,107 @@
+"""hangman, letter guessing game
+
+Exercises
+1. Change Hangman's picture (position, length, etc)
+2. Make a additional dictionary, not a fruit
+3. Convert lowercase letters to uppercase letters
+4. Make it possible to enter not only the alphabet, but also the word to guess the answer
+"""
+
+from turtle import *
+from random import choice
+
+def screen_write(letter):
+    """write text on the screen"""
+    write(letter, move=True, font=("Courier", 14, "normal"))
+
+def move_write(x, y, letter):
+    """After move, write text on the screen"""
+    goto(x, y)
+    screen_write(letter)
+
+
+def draw(x, y, angle, length, straight):
+    """draw a hangman game"""
+    goto(x, y)
+    pendown()
+    left(angle)
+    if straight == True:
+        forward(length)
+    elif straight == False:
+        circle(length, None, 100)
+    penup()
+
+def drawHangman(num):
+    if num == 1: 
+        """draw head"""
+        draw(-45, 160, 270, 15, False)
+    elif num == 2:
+        """draw torso"""
+        draw(-45, 130, 90, 40, True)
+    elif num == 3:
+        """draw left arm"""
+        draw(-45, 120, 235, 40, True)
+    elif num == 4:
+        """draw right arm"""
+        draw(-45, 120, 250, 40, True)
+    elif num == 5:
+        """draw left leg"""
+        draw(-45, 90, 205, 40, True)
+    elif num == 6:
+        """draw right leg"""
+        draw(-45, 90, 60, 40, True)
+
+hideturtle()
+speed(0)
+
+fruitword = ['apple', 'avocado', 'banana', 'blueberry', 'cherry', 'citron',
+        'coconut', 'durian', 'fig', 'grape', 'grapefruit', 'kiwi', 'lemon',
+        'lime', 'mandarin', 'mango', 'melon', 'orange', 'papaya', 'peach',
+        'pear', 'persimmon', 'pineapple', 'plum', 'raspberry', 'strawberry',
+        'tomato', 'watermelon']
+
+"""draw gallows"""
+penup()
+draw(50, 0, 90, 175, True)
+draw(50, 175, 90, 95, True)
+draw(-45, 175, 90, 15, True)
+
+goto(-100,-35)
+word = choice(fruitword)
+for i in word:
+    screen_write("_ ")
+correct = []
+use = []
+wrong = 0
+while wrong < 6:
+    """Modify the code to solve Exercise 4"""
+    letter = textinput("Hangman","Guess a letter:")
+    while len(letter) != 1:
+        letter = textinput("Hangman","Guess a letter:")
+    goto(-100,-35)
+    if letter not in use:
```

Comment:
Remove this check. Allow someone to guess the same letter twice. It's dumb but it'll just lose a turn.